echemdb/data/cv/entry.py

```diff
@@ -380,48 +337,89 @@ def __repr__(self):
         """
         return f"Entry({repr(self.identifier)})"
 
-    def plot(self, xunit=None, yunit=None):
+    def _verify_field_name(self, field_name):
+        """Verify if a given field name exists in the resources field names.
+
+        EXAMPLES::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry._verify_field_name('E')
+            'E'
+
+        For a field with name `x` that does not exist::
+
+            >>> entry = Entry.create_examples()[0]
+            >>> entry._verify_field_name('x')
+            Traceback (most recent call last):
+            ...
+            ValueError: None of the axes is named 'x'.
+
+        """
+        if field_name not in self.field_names:
+            raise ValueError(f"None of the axes is named '{field_name}'.")
+        return field_name
+
+    def plot(self, x_label='E', y_label='j', units={}):
```

Comment:
Probably we do not want to have `units={}` here. Instead, I guess we want to force the following:
entry.rescale(units={...}).plot()

Reply:
adapted in the recent changes